icu4c/source/common/lstmbe.h

```diff
@@ -68,7 +68,7 @@ class LSTMBreakEngine : public DictionaryBreakEngine {
     const Vectorizer* fVectorizer;
 };
 
-U_CAPI const LanguageBreakEngine* U_EXPORT2 CreateLSTMBreakEngine(
```

Comment:
You cannot remove the const here. The LanguageBreakEngine have to be immutable and the internal state cannot be changed after construction.

Reply:
Fixed